src/web/components/ParticipantRequests/ParticipantRequestForm.stories.tsx

```diff
@@ -20,6 +20,7 @@ const response: SiteDTO[] = [
     clientTypes: ['PUBLISHER'],
     // eslint-disable-next-line camelcase
     client_count: 1,
+    visible: true,
```

Comment:
It doesn't matter what value `visible` is in the participant request form, so I just did one with `true` and one with `false`